### PR TITLE
Add some information about how we can review and test PR quickly

### PR DESCRIPTION
**Pull request by AhyoungRyu**
### What is this PR for?
Zeppelin has already provided a useful utility for reviewing PR ( #513 ). So I added a **Testing a Pull Request** section to `CONTRIBUTING.md` for whom to review and test PR quickly.

### What type of PR is it?
Documentation

### Todos
* [x] - Add a **Testing a Pull Request** section to CONTRIBUTING.md  

### Is there a relevant Jira issue?
[ZEPPELIN-484](https://issues.apache.org/jira/browse/ZEPPELIN-484) and PR #513 

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -39,6 +39,29 @@ You can also use this small bookmarklet tool to fill your Pull Request fields au
...
+## Testing a Pull Request
+You can also test and review a particular Pull Request. Here are two useful ways.
+
+* Using a utility provided from Zeppelin. 
+    
+    ```
+    dev/test_zeppelin_pr.py [# of PR]
+    ```
+
+    For example, if you want to test `#513`, then the command will be like:
```
Comment (corneadoug) [inline CONTRIBUTING.md]: then the command will be like:
-> then the command will be:

Comment (AhyoungRyu) [inline CONTRIBUTING.md]: @corneadoug applied it : )

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -39,6 +39,29 @@ You can also use this small bookmarklet tool to fill your Pull Request fields au
...
+    dev/test_zeppelin_pr.py 513
+    ```
+
+* Another way is using [github/hub](https://github.com/github/hub). 
+     
+    ```
+    hub checkout https://github.com/apache/incubator-zeppelin/pull/[# of PR]
+    ```
+
+The above two methods will help you to test and review the Pull Request.
```
Comment (felixcheung) [inline CONTRIBUTING.md]: The above two methods will help you to test and review the Pull Request.
->
The above two methods will help you test and review Pull Requests.
?

Comment (AhyoungRyu) [inline CONTRIBUTING.md]: @felixcheung I think that "help you to test" is same with "help you test", isn't it? : )

Comment (felixcheung) [inline CONTRIBUTING.md]: I guess `to` is optional after `help`, but it seems more common without
https://www.google.com/search?q=will+help+to
anyway, this is nit picking. ;)

Comment (AhyoungRyu) [inline CONTRIBUTING.md]: @felixcheung Make sense. I applied and committed it : )

**Comment by felixcheung** [comment]
any other comment? let's merge this?

**Comment by jongyoul** [comment]
LGTM. Let's merge it.

**Comment by Leemoonsoo** [comment]
LGTM

**Comment by felixcheung** [comment]
merged. thx

**Pull request closed without merging**